### ISSUE TITLE
Notebook will not update from 6.5.2 to 7 on Mac

### ISSUE THREAD

**Issue by lpmathiasendk** [jupyter/notebook#7310]
Please answer the following questions for yourself before submitting an issue
- [ x ] I checked the documentation and found no answer
- [ x ] I checked to make sure that this issue has not already been filed
- [ x ] I'm reporting the issue to the correct repository

If you have further questions after reading below, please visit the Jupyter Notebook discourse channel (https://discourse.jupyter.org/) and submit your questions there. There are many more people in the Jupyter community that engage on that channel.

NOTE:
Jupyter Notebook 6.x development is in maintenance-only mode. Bugs found in Notebook 6.x and that don't reproduce in Notebook 7.x may not get fixed.

Work in this repository is focused on Jupyter Notebook 7.x, the former RetroLab project. The code base for Notebook 7.x is very different than Notebook 6.x. If you can, please try reproducing Notebook 6.x bugs with Notebook 7.x using the main branch of this repository.

We recommend that you check out JupyterLab (https://github.com/jupyterlab/jupyterlab), Jupyter's next generation Notebook interface.

Here, we're looking for specific bugs in the Jupyter Notebook codebase. If you think you've identified such a bug, you can continue opening your issue here. We'd appreciate if you include as much detail as possible, such as links to the offending code, snapshots of the UI issue, code blocks with your console logs, etc.
-->

## Description
I just installed Jupyter Notebook on my Mac Intel today but I get version 6.5.2 and no matter what I do I'm not getting the version 7. 

## Reproduce
I used `pip install -U jupyter` and `conda update jupyter` from https://docs.jupyter.org/en/latest/use/upgrade-notebook.html#upgrading-jupyter-notebook-using-anaconda

## OS
I am on a Mac Intel i9 with MacOS 14.4.1 (23E224)

**Comment by lpmathiasendk** [comment]
Yes. I never succeded in updating from 6 to 7. VenligstLars MathiasenDen 2. apr. 2024 kl. 18.32 skrev Rosio ***@***.***>:﻿
Hi @lpmathiasendk, have you tried updating Notebook specifically with pip install notebook or conda update notebook? If so, we might need to update the documentation as the documentation you linked might be out of date. Please let us know and I can open another issue for that in the appropriate repository.

**Comment by JasonWeill** [comment]
As an alternative, if possible, you could try installing `notebook` with the `conda-forge` repo, using `conda install -c conda-forge notebook`. Does this provide Notebook 7?
